jenkins/databricks/build.sh

```diff
@@ -127,6 +128,6 @@ if [ `ls $DB_JAR_LOC/cudf* | wc -l` -gt 1 ]; then
     ls $DB_JAR_LOC/cudf*
     exit 1
 fi
-$SPARK_HOME/bin/spark-submit ./runtests.py --runtime_env="databricks"
+. run_pyspark_from_build.sh --runtime_env="databricks"
```

Comment:
Is this supposed to be `./run_pyspark_from_build.sh`?  This script isn't designed to be sourced, and it's confusing to see it treated that way.  I'd rather see it executed directly or used via something like `bash -c`.

Reply:
changed to `bash run_pyspark_from_build.sh --runtime_env="databricks"`